.github/workflows/manual-deploy-obscuroscan.yml

```diff
@@ -0,0 +1,71 @@
+# Deploys Obscuroscan on Azure for Testnet
+# Builds the Obscuroscan network image, pushes the image to dockerhub and starts the Obscuroscan on azure
+#
+# Obscuroscan is a docker container that runs the obscuroscan package
+# It exposes the following ports:
+# HTTP:       80,
+#
+# Exposes the following addresses: (only accessible internally)
+#  testnet-obscuroscan.uksouth.azurecontainer.io
+
+name: Manual Deploy Obscuroscan
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+
+# to be used when our dockerhub account is fixed
+#      - name: Login to DockerHub
+#        uses: docker/login-action@v2
+#        with:
+#          username: ${{ secrets.DOCKERHUB_USERNAME }}
+#          password: ${{ secrets.DOCKERHUB_TOKEN }}
+#
+#      - name: Build and push
+#        uses: docker/build-push-action@v3
+#        with:
+#          push: true
+#          tags: obscuronet/obscuro_testnet_l1network:latest
+#          file: ./testnet/gethnetwork.Dockerfile
+
+      - name: 'Login via Azure CLI'
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: 'Build and push image'
+        uses: azure/docker-login@v1
+        with:
+          login-server: ${{ secrets.REGISTRY_LOGIN_SERVER }}
+          username: ${{ secrets.REGISTRY_USERNAME }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+      - run: |
+          docker build -t ${{ secrets.REGISTRY_LOGIN_SERVER }}/obscuronet/obscuro_testnet_obscuroscan:latest -f testnet/obscuroscan.Dockerfile  .
+          docker push ${{ secrets.REGISTRY_LOGIN_SERVER }}/obscuronet/obscuro_testnet_obscuroscan:latest
+
+      - name: 'Deploy to Azure Container Instances'
+        uses: 'azure/aci-deploy@v1'
+        with:
+          resource-group: ${{ secrets.RESOURCE_GROUP }}
+          dns-name-label: testnet-obscuroscan
+          image: ${{ secrets.REGISTRY_LOGIN_SERVER }}/obscuronet/obscuro_testnet_obscuroscan:latest
+          registry-login-server: ${{ secrets.REGISTRY_LOGIN_SERVER }}
+          registry-username: ${{ secrets.REGISTRY_USERNAME }}
+          registry-password: ${{ secrets.REGISTRY_PASSWORD }}
+          name: testnet-obscuroscan
+          location: 'uksouth'
+          restart-policy: 'Never'
+          command-line: ./tools/obscuroscan/main/main --rpcServerAddress testnet.obscu.ro:13000 --address 0.0.0.0:80
```

Comment:
Do you need a sudo to bind port 80?

Reply:
Good question - It runs as super user afaik